chain/network/src/types.rs

```diff
@@ -809,3 +950,29 @@ mod tests {
         assert_size!(NetworkInfo);
     }
 }
+
+#[derive(Message, Clone, Serialize, Deserialize)]
+#[rtype(result = "()")]
+pub struct SetRoutingTableRequest {
+    pub add_edges: Option<Vec<Edge>>,
+    pub remove_edges: Option<Vec<SimpleEdge>>,
+    pub prune_edges: Option<bool>,
+}
+
+#[derive(Serialize)]
+pub struct RpcError(pub String);
```

Comment:
I feel this is not the appropriate place to define JSON RPC specific types. I believe near-network should not be aware of JSON RPC.

Reply:
@frol Ok, I'll two different data structures for that purpose.